lib/3_add_up_numbers.js

```diff
@@ -6,6 +6,9 @@
 
 module.exports = {
   add_up_numbers: function(arr){
-    // Your code goes here.
+    var total = arr.reduce(function(x, y) {
```

Comment:
Please be mindful of indentation! _Within_ (ie indented one space more than) the outermost function, which begins on line 8, I'd expect to see indentation like so:

```javascript
var total = arr.reduce(function(x, y) {
  return x + y;
});
return_total;